src/Aggregate.ts

```diff
@@ -81,6 +80,7 @@ export default abstract class Aggregate<StateType> {
     if (!event.id) {
       event.id = uuid();
     }
+    // eslint-disable-next-line
```

Comment:
Boo! Disabling instead of changing the selection to strict equality check.

Reply:
We don't want strict equality check here. How does one override the eslint rules ?